src/containers/message/side-bar/ui.tsx

```
@@ -1,132 +1,296 @@
+				updateQuery: (prev, { fetchMoreResult }) => {
+					if (!fetchMoreResult || fetchMoreResult?.getRooms?.length === 0) {
+						setIsDone(true);
+						return prev;
+					}
+					// 전역 상태에 페이지 업데이트
+					setPage(newPage);
+					return {
+						...prev,
+						getRooms: [...(prev?.getRooms ?? []), ...fetchMoreResult?.getRooms],
+					};
+				},
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)